packages/react-core/src/components/Modal/examples/Modal.md

```diff
@@ -383,3 +382,107 @@ class NoHeader extends React.Component {
   }
 }
 ```
+
+```js title=No-title
```

Comment:
the no header and no title examples seem identical?

Reply:
Lol, yep. That's evident of me experiencing tunnel vision.